courses/models.py

```diff
@@ -172,6 +172,20 @@ def current_price(self):
             return None
         return latest_version.price
 
+    @property
+    def unexpired_run_dates(self):
+        """
+        Gets all the start dates for unexpired CourseRuns in all the Courses associated with this program
+
+        Returns:
+            [Datetime]: list of start_dates from all unexpired course runs of all courses in this program
+        """
+        courses = self.courses.all()
+        dates = []
+        for course in courses:
+            dates.append(course.unexpired_run_dates())
+        return dates
```

Comment:
this method is never used ?
Also if we need unexpired runs for a program, may be we should return dict against each course run.
e.g
```
unexpired_course_runs = {
'course_run_id': [datetime, datetime],
...
}
```

Reply:
It will be used on the Program page, which is not developed yet. Acceptance criteria on the original ticket (#161 ) does not require course info, only dates, that's why I see no need to include course-wise dates, a flat list would do.

Reply:
Still, I think it would makes sense to add a dictionary, probably we would need to filter on courses on front end. Flat list of dates does not make sense.

Reply:
Can you please tag Peter or Ferdi here or on Slack to get clarification on this? As far as the original ticket is concerned, this is going to be a flat list of start dates of unexpired course runs for all courses in the program.

Reply:
@pdpinch Your thoughts on this ?